bundle-workflow/tests/signing_workflow/__init__.py

```diff
@@ -0,0 +1,11 @@
+#  SPDX-License-Identifier: Apache-2.0
+#
+#  The OpenSearch Contributors require contributions made to
+#  this file be licensed under the Apache-2.0 license or a
+#  compatible open source license.
+
+import os
+import sys
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "../.."))
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "../../src"))
```

Comment:
idk if there is a more elegant way to define this, but without /src it could not find the signer's import of git_repository.

Reply:
I couldn't find a better way to do it.

You should have just one though, and in tests import `from src.git.git_repository import GitRepository`, note the `src.`. As written you may have a conflict in class names between `tests` and `src`.